output/slide-deck.qmd

```diff
@@ -238,9 +255,10 @@ will see later on what these files are)
 <br>
 
 ::: nonincremental
-1.  **In your GitHub repo** click on the "Code" button
+1.  **In your GitHub repo**, return to the "Code" tab and click on the green
+    "Code" button
 
-2.  In the "HTTPS" tab, click on the "Copy" button
```

Comment:
They might not know what a 'copy' button is?

Reply:
Ok! Yep, that's fine then

Reply:
Solved in ebf35474652fc7aebcbed66c3312cd862f76137e